core/src/main/scala/kafka/zk/KafkaZkClient.scala

```diff
@@ -569,18 +570,27 @@ class KafkaZkClient private[zk] (zooKeeperClient: ZooKeeperClient, isSecure: Boo
   }
 
   /**
-   * Gets the assignments for the given topics.
+   * Gets the replica assignments for the given topics.
```

Comment:
Same thing here.  We should explain that this doesn't handle reassigning replicas